### ISSUE TITLE
Reshape not acting as (I) expected

### ISSUE THREAD

**Issue by holmescharles** [johnkerl/miller#1032]
Let's say I run the following command:

```sh
mlr --pprint reshape -s item,value << EOF
    time       item value
    2009-01-01 X    0.65473572
    2009-01-01 Y    2.4520609
    2009-01-02 X    -0.89248112
    2009-01-03 X    0.98012375
    2009-01-03 Y    1.3179287
EOF
```

I would expect to get out something like:

```
    time       X           Y
    2009-01-01 0.65473572  2.4520609
    2009-01-02 -0.89248112 (error)
    2009-01-03 0.98012375  1.3179287
```

Instead, I get:

```
 time       X          Y
 2009-01-01 0.65473572 2.4520609

 time       X
 2009-01-02 -0.89248112

 time       X          Y
 2009-01-03 0.98012375 1.3179287
```

Two questions:
1. Is the latter output (the actual output) expected, or is this a bug?
2. Is there a way to get output as I expected (one table with empty values handled some how)? Ideally, I'd like to supply it with some fill value (-99999).

Thanks!

**Comment by holmescharles** [comment]
I found the verb unsparsify, which gets me most of where I want to go. Now, I have this bug:

The following output prints the usage message (i.e., fails).

```sh
mlr --pprint reshape -s item,value then unsparsify --fill-with -99999 << EOF
    time       item value
    2009-01-01 X    0.65473572
    2009-01-01 Y    2.4520609
    2009-01-02 X    -0.89248112
    2009-01-03 X    0.98012375
    2009-01-03 Y    1.3179287
EOF
```

**Comment by aborruso** [comment]
@johnkerl the same for me, it seems related to negative number.

The below command works in Miller 5, but not in 6.

```
mlr  --csv reshape -s item,value then unsparsify --fill-with -9999 input.csv
```

And we have the same problem using `fill-empty` verb.

```
time,item,value
2009-01-01,X,0.65473572
2009-01-01,Y,2.4520609
2009-01-02,X,-0.89248112
2009-01-03,X,0.98012375
2009-01-03,Y,1.3179287
```

**Comment by johnkerl** [comment]
@holmescharles @aborruso sorry for the delay (on this and all things Miller-related) the last couple months.

This is fixed on #1048.

**Comment by johnkerl** [comment]
Closing given #1048 -- please let me know if I missed something & we can re-open. Thank you! :)
